web/index.html

```diff
@@ -170,6 +173,10 @@ <h6 class="text-secondary">(<span id="template-pixel-right-percentage"
                         document.querySelector('#place-capable').innerText = payload.capabilities.place;
                         document.querySelector('#pixels-per-minute').innerText = Math.round(payload.capabilities.place / 5);
 
+                        const timeLeft = payload.completion.wrong / (payload.capabilities.place / 5);
+                        document.querySelector('#time-hours-left').innerText = Math.floor(timeLeft / 60);
+                        document.querySelector('#time-minutes-left').innerText = Math.floor(timeLeft % 60);
```

Comment:
Completion stats are optional (reddit integration can be turned off), see the if block below. The ETA should be hidden if we don't know completion.